backends-common/cassandra/pom.xml

```diff
@@ -168,6 +168,12 @@
                     <version>${assertj-3.version}</version>
                     <scope>test</scope>
                 </dependency>
+                <dependency>
+                    <groupId>nl.jqno.equalsverifier</groupId>
+                    <artifactId>equalsverifier</artifactId>
+                    <version>1.7.6</version>
+                    <scope>test</scope>
+                </dependency>
```

Comment:
Indent issue!
I don't know why but we have often indent issue with pom.xml I guess it's a mix between space and tab.
When editing pom.xml make sure to reindent the all file